skglm/solvers/cd_solver.py

```diff
@@ -31,7 +31,10 @@ def cd_solver_path(X, y, datafit, penalty, alphas=None,
     alphas : ndarray
         List of alphas where to compute the models.
 
-    coef_init : ndarray, shape (n_features,) | None, optional, (default=None)
+    fit_intercept : bool
+        Whether or not to fit an intercept.
+
+    coef_init : ndarray, shape (n_features + 1,) | None, optional, (default=None)
```

Comment:
n_features + 1 even if no intercept is used?

I'd be explicit: `shape (n_features,) or (n_features + 1,)`.